<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>INIT</class>
 <widget class="QWidget" name="INIT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="MYBTN" name="btn_start">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Script</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Segoe Script&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MYBTN" name="btn_min">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MYBTN" name="btn_close">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>0</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYBTN</class>
   <extends>QPushButton</extends>
   <header location="global">mycontral/mybtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
